mappings/net/minecraft/network/MessageDecorator.mapping

```diff
@@ -1,28 +1,29 @@
-	COMMENT {@link net.minecraft.server.MinecraftServer#getChatDecorator}.
+CLASS net/minecraft/class_7492 net/minecraft/network/MessageDecorator
+	COMMENT Message decorator decorates the chat messages and other messages server-side.
+	COMMENT Currently, only one message decorator can exist at a time. The message decorator
+	COMMENT that is currently used can be obtained by
+	COMMENT {@link net.minecraft.server.MinecraftServer#getMessageDecorator}.
 	COMMENT
-	COMMENT <p>For the chat decorator to produce a signed message, <strong>both the server
+	COMMENT <p>For the message decorator to produce a signed message, <strong>both the server
 	COMMENT and the sender's client need to have chat previews enabled</strong>, Otherwise, the decorated
 	COMMENT content is considered unsigned, and if the clients require chat messages to be signed
 	COMMENT via the {@linkplain net.minecraft.client.option.GameOptions#getOnlyShowSecureChat
 	COMMENT "Only Show Secure Chat" option}, they will see the undecorated message. Therefore,
-	COMMENT chat decorator is <strong>not recommended for censoring messages</strong>.
+	COMMENT message decorator is <strong>not recommended for censoring messages</strong>.
 	COMMENT
 	COMMENT <p>It is <strong>very important that the decorator return the same text when previewed
-	COMMENT and sent</strong>. If this is not followed correctly, the server detects that the client
-	COMMENT sent a forged text and discards the message. For example, a decorator that appends the
-	COMMENT time the decoration was applied would be likely to fail, since there is usually a delay
-	COMMENT between the previewing and the submission. One way to solve this issue is to make it
-	COMMENT cache the result on preview, so that when the sent message needs decorating, the cached
-	COMMENT value can be used.
+	COMMENT and sent</strong>. If this is not followed correctly, the server will discard the message.
+	COMMENT For example, a decorator that appends the time the decoration was applied would be
+	COMMENT likely to fail, since there is usually a delay between the previewing and the
+	COMMENT submission. One way to solve this issue is to make it cache the result on preview,
+	COMMENT so that when the sent message needs decorating, the cached value can be used.
 	FIELD field_39384 NOOP Lnet/minecraft/class_7492;
-		COMMENT An empty chat decorator that does not decorate anything.
+		COMMENT An empty message decorator that does not decorate anything.
 	METHOD decorate (Lnet/minecraft/class_3222;Lnet/minecraft/class_2561;)Ljava/util/concurrent/CompletableFuture;
-		COMMENT {@return the signed chat message with unsigned content set as decorated {@code message}}
+		COMMENT {@return the decorated {@code message}}
 		COMMENT
-		COMMENT <p>If the received player requires signed chat message, they will see the original content.
+		COMMENT @apiNote Custom message decorators should implement this method (usually as
```

Comment:
This note is quite useless as we see this is already an abstract method. I suggest removing this section.